101-cosmosdb-webapp/azuredeploy.json

```diff
@@ -0,0 +1,135 @@
+{
+"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "applicationName": {
+      "type": "string",
+      "defaultValue": "[concat('to-do-app', uniqueString(resourceGroup().id))]",
+      "metadata": {
+        "description": "Application Name, max length 30 characters"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "appServicePlanTier": {
+      "type": "string",
+      "defaultValue": "F1",
+      "allowedValues": [
+        "F1",
+        "D1",
+        "B1",
+        "B2",
+        "B3",
+        "S1",
+        "S2",
+        "S3",
+        "P1",
+        "P2",
+        "P3",
+        "P4"
+      ],
+      "metadata": {
+        "description": "App Service Plan's pricing tier. Details at https://azure.microsoft.com/en-us/pricing/details/app-service/"
+      }
+    },
+    "appServicePlanInstances": {
+      "type": "int",
+      "defaultValue": 1,
+      "minValue": 1,
+      "maxValue": 3,
+      "metadata": {
+        "description": "App Service Plan's instance count"
+      }
+    },
+    "repoURL": {
+      "type": "string",
+      "defaultValue": "https://github.com/Azure-Samples/cosmos-db-to-do.git",
+      "metadata": {
+        "description": "The URL for the GitHub repository that contains the project to deploy."
+      }
+    },
+    "branch": {
+      "type": "string",
+      "defaultValue": "master",
+      "metadata": {
+        "description": "The branch of the GitHub repository to use."
+      }
+    }
+  },
+  "variables": {
+    "cosmosAccountName": "[toLower(parameters('applicationName'))]",
+    "webSiteName": "[parameters('applicationName')]",
+    "hostingPlanName": "[parameters('applicationName')]",
+    "repoUrl": "[parameters('repoURL')]",
+    "branch": "[parameters('branch')]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.DocumentDB/databaseAccounts",
+      "name": "[variables('cosmosAccountName')]",
+      "apiVersion": "2020-04-01",
+      "kind": "GlobalDocumentDB",
+      "location": "[parameters('location')]",
+      "properties": {
+        "consistencyPolicy": {"defaultConsistencyLevel": "Session"},
+        "locations": [{"locationName": "[parameters('location')]", "failoverPriority": 0, "isZoneRedundant": false}],
```

Comment:
format the JSON (e.g. SHIFT+ALT+F in VS Code)

Reply:
ok, fixed in next commit.